### ISSUE TITLE
DPLMS implementation

### ISSUE THREAD

**Issue by theHenks** [legend-exp/LegendDSP.jl#7]
Need Valerio here for some support

**Comment by theHenks** [comment]
@johannaluehrs I was told you are working on this? Maybe we can use this issue to track progress on the DPLMS implementation. :) 

CC @wiesinger-ch

**Comment by johannaluehrs** [comment]
Yes sure! How should I track this here? I don't have that much experience with communication on GitHub :)

**Comment by theHenks** [comment]
Just comment under this thread. In case you encounter problems, or already have a first implementation on a branch, it is easiest to use this here for communication around the implementation.
